packages/apps/hr/src/components/atoms/ExpenseLineCard/ExpenseLineCard.tsx

```diff
@@ -0,0 +1,150 @@
+/*
+ * Axelor Business Solutions
+ *
+ * Copyright (C) 2023 Axelor (<http://axelor.com>).
+ *
+ * This program is free software: you can redistribute it and/or  modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React, {useEffect, useMemo, useRef} from 'react';
+import {Animated, StyleSheet, TouchableOpacity, View} from 'react-native';
+import {
+  Card,
+  Text,
+  checkNullString,
+  useThemeColor,
+  Checkbox,
+} from '@axelor/aos-mobile-ui';
+import {useSelector} from '@axelor/aos-mobile-core';
+
+interface ExpenseLineCardProps {
+  style?: any;
+  expenseDate?: string;
+  projectName?: string;
+  totalAmount?: string;
+  displayText?: string | number;
+  onPress: () => void;
+  onLongPress: () => void;
+  onItemSelection: () => void;
+  isSelectionMode?: boolean;
+}
+
+const ExpenseLineCard = ({
+  style,
+  expenseDate,
+  projectName,
+  totalAmount,
+  displayText,
+  onPress,
+  onLongPress,
+  isSelectionMode,
+  onItemSelection,
+}: ExpenseLineCardProps) => {
+  const Colors = useThemeColor();
+
+  const {user} = useSelector((state: any) => state.user);
+
+  const translateXAnim = useRef(new Animated.Value(0)).current;
+
+  useEffect(() => {
+    if (isSelectionMode) {
+      Animated.timing(translateXAnim, {
+        toValue: 30,
+        duration: 300,
+        useNativeDriver: true,
+      }).start();
+    } else {
+      Animated.timing(translateXAnim, {
+        toValue: 0,
+        duration: 300,
+        useNativeDriver: true,
+      }).start();
+    }
+  }, [isSelectionMode, translateXAnim]);
+
+  const styles = useMemo(() => getStyles(Colors), [Colors]);
+
+  return (
```

Comment:
Please improve design of card, margin is not equal on both side on the card in the list.

This can be a idea but do not hesitate to improve it, we want it to like nice.
![Capture d’écran du 2023-08-22 10-15-34](https://github.com/axelor/axelor-mobile/assets/102581501/127f595e-8c62-40b6-83d9-bf03310dffc3)